aspnetcore/signalr/cpp-client.md

```diff
@@ -0,0 +1,100 @@
+---
+title: ASP.NET Core SignalR C++ Client
+author: bradygaster
+description: Information about the ASP.NET Core SignalR C++ Client
+monikerRange: '>= aspnetcore-3.0'
+ms.author: bradyg
+ms.custom: mvc
+ms.date: 2/26/2019
+uid: signalr/cpp-client
+---
+
+# ASP.NET Core SignalR C++ Client
+
+The ASP.NET Core SignalR C++ client library lets you communicate with SignalR hubs from native applications.
```

Comment:
Might we add some verbiage here about what benefits that'd offer? Like "why would you care," as a reader?

Reply:
Idk, I feel like if you're reading a doc on a C++ client, you should already be aware of the usefulness to you. Otherwise you wouldn't have clicked the link.